libtailscale/localapi.go

```diff
@@ -18,9 +19,14 @@ import (
 	"strings"
 	"sync"
 	"time"
+
+	"tailscale.com/ipn"
 )
 
-// CallLocalAPI calls the given endpoint on the local API using the given HTTP method
+const timeoutMillis = 30000
```

Comment:
On the Kotlin side, we specify the timeout on a per request basis, we just happen to default to 30000 millis. If this is really just the timeout for the `EditPrefs` call, it might make sense to hardcode that inside of `EditPrefs`.

Reply:
use time.Duration

const timeout = 30 * time.Second